bookwyrm/templates/book/book.html

```diff
@@ -8,28 +8,35 @@
 <div class="block" itemscope itemtype="https://schema.org/Book">
     <div class="columns is-mobile">
         <div class="column">
-            <h1 class="title">
-                <span itemprop="name">
-                    {{ book.title }}{% if book.subtitle %}:
-                        <small>{{ book.subtitle }}</small>
+            <h1 class="title" itemprop="name">
+                {{ book.title }}
+            </h1>
+
+            {% if book.subtitle or book.series %}
+                <p class="subtitle">
```

Comment:
I agree with @joachimesque about the font weight - I think the subtitle should be visually close to the title, rather than the author.

Reply:
I’m confused with this comment because that’s what the PR does. :)

Reply:
Sorry for being unclear! I just mean that when the subtitle has a lighter font weight (as here: https://user-images.githubusercontent.com/277773/119652123-03664200-be26-11eb-85ec-a25899ada82d.png) it looks visually similar to the author, as opposed to with a heavier font weight (as here:https://user-images.githubusercontent.com/277773/121321562-813e4900-c90e-11eb-820a-3d2cc21ae520.png), where the title and subtitle are more visually similar.